packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -29,7 +29,14 @@ class MultiChildLayoutParentData extends ContainerBoxParentData<RenderBox> {
 /// given the previous instance.
 ///
 /// Override [getSize] to control the overall size of the layout. The size of
-/// the layout cannot depend on layout properties of the children.
+/// the layout cannot depend on layout properties of the children because
```

Comment:
Maybe simplify this long sentence a little: "... properties of the children. This was a design decision to simplify the delegate implementations: This way, the delegate implementations don't have to also handle various intrinsic sizing functions if the parent's size depended on the children."